src/meta/src/manager/catalog/mod.rs

```diff
@@ -2147,6 +2147,10 @@ where
         self.core.lock().await.database.get_table_name_mapping()
     }
 
+    pub async fn get_table_type_mapping(&self) -> HashMap<TableId, String> {
```

Comment:
We don't need a separate fn to acquire the lock the fetch the table type mapping. We can just modify `get_table_name_mapping` to fetch the table name and type in one shot given that this function is only used in emitting the table_info metric.